### DISCUSSION TITLE
Can't return Span from function

### DISCUSSION THREAD

**Discussion by Xyncgas** [fsharp/fslang-suggestions#1169] [General]
![image](https://user-images.githubusercontent.com/35312807/185514435-b18f7c77-92c2-4af1-a8aa-90b484f6d309.png)

**Comment by vzarytovskii** [comment]
> ![image](https://user-images.githubusercontent.com/35312807/185514435-b18f7c77-92c2-4af1-a8aa-90b484f6d309.png)
> 

This repository is for language suggestions only. You will get your question answered faster on some specialized resources, like StackOverflow, F# slack or subreddit.
